NeuroID/NeuroIDClass/NeuroID.swift

```diff
@@ -104,21 +103,21 @@ public enum NeuroID {
 
     public static func stop() {
         NIDPrintLog("NeuroID Stopped")
-        setUserDefaultKey(Constants.storageLocalNIDStopAllKey.rawValue, value: true)
         do {
             _ = try closeSession(skipStop: true)
         } catch {
             NIDPrintLog("NeuroID Error: Failed to Stop because \(error)")
         }
 
+        NeuroID.groupAndPOST()
```

Comment:
Does this do event flush?

Reply:
Correct, that is the function that sends all the events